samples/subsys/rtio/sensor_batch_processing/src/main.c

```diff
@@ -35,8 +34,7 @@ void main(void)
 	for (int n = 0; n < N; n++) {
 		struct rtio_sqe *sqe = rtio_spsc_acquire(ez_io.sq);
 
-		rtio_sqe_prep_read(sqe, iodev, RTIO_PRIO_HIGH, bufs[n],
-				   SAMPLE_SIZE, bufs[n]);
+		rtio_sqe_prep_read(sqe, iodev, RTIO_PRIO_HIGH, NULL, 0, NULL);
```

Comment:
Probably need a read_with_pool() prep, can take 3 params (sqe, iodev, prio); and be done

Reply:
Needed `userdata` too, but done